pkg/prom/result.go

```diff
@@ -166,7 +181,7 @@ func NewQueryResults(query string, queryResult interface{}) (*QueryResults, erro
 	return &QueryResults{
```

Comment:
Return `qrs` here! 👍

Reply:
Oh yep

Reply:
False alarm I think. Looks like you return qrs every error, although might be possible to just attach results to `qrs` and return.